### PR TITLE
SC-2256: Adjust Travis not to generate assets

### PR DESCRIPTION
**Pull request by alexanderM91**
#### Overview

- Ticket: https://spryker.atlassian.net/browse/SC-2197

- Suite-nonsplit PR: https://github.com/spryker/suite-nonsplit/pull/3029

- Release Group: https://release.spryker.com/release-groups/view/2017

##### Improvements

- Introduced a loading of the demo data by running the specified section from the docker installation recipe for each store.
- Adjust Travis performance for Docker. Execution per store and reduced operation steps, e.g assets building.

### CODE REVIEW

File: ci/deploy.yml [hunk 1]
```diff
@@ -7,82 +7,109 @@ environment: docker
 image: spryker/php:7.2
```
Comment (sprymiker) [inline ci/deploy.yml]: this file is used for tests for docker sdk. Please use a file on project level.

File: generator/src/templates/deploy.bash.twig [hunk 2]
```diff
@@ -174,22 +174,32 @@ function areAssetsBuilt()
 {% endif %}
```
Comment (sprymiker) [inline generator/src/templates/deploy.bash.twig]: Extend help of `demo-data` command that tells about argument

File: generator/src/templates/deploy.bash.twig [hunk 3]
```diff
@@ -425,18 +435,18 @@ function loadDemoData()
         export SPRYKER_STORE="${STORES[0]}"
         echo -e "${INFO}Loading demo data for ${SPRYKER_STORE} store.${NC}"
         initDatabase
-        execSpryker 'vendor/bin/install -r docker -s init-storages-per-region -s demodata'
+
+        local demoDataSection=${1:-demodata}
```
Comment (sprymiker) [inline generator/src/templates/deploy.bash.twig]: Also first argument is already occupied by `if [ "$1" = ${IF_NOT_PERFORMED} ]`. Use second one.

File: generator/src/templates/deploy.bash.twig [hunk 4]
```diff
@@ -637,7 +659,7 @@ case ${command} in
         ;;
     demo|demo-data)
         runApp
-        loadDemoData
+        loadDemoData ${@}
```
Comment (sprymiker) [inline generator/src/templates/deploy.bash.twig]: loadDemoData "" ${1}

File: ci/deploy.yml (hunk 1 above)
Comment (alexanderM91) [reply ci/deploy.yml]: Introduced a new `docker.ci.yml`

**Pull request merged**